driver-core/src/main/com/mongodb/internal/connection/TlsChannelStreamFactoryFactory.java

```diff
@@ -71,13 +72,18 @@ public class TlsChannelStreamFactoryFactory implements StreamFactoryFactory {
     /**
      * Construct a new instance
      */
-    public TlsChannelStreamFactoryFactory(final InetAddressResolver inetAddressResolver) {
+    public TlsChannelStreamFactoryFactory(final InetAddressResolver inetAddressResolver,
```

Comment:
This new constructor does not need to be `public`.

```suggestion
    TlsChannelStreamFactoryFactory(final InetAddressResolver inetAddressResolver,
```